src/components/Header/index.js

```diff
@@ -0,0 +1 @@
+export * from "./Header.jsx";
```

Comment:
should not have the extension at the end the packager will just figure it out and it makes it easier for you to switch to typescript (tsx) without changing much